record_adb.py

```diff
@@ -0,0 +1,48 @@
+import sys
+import argparse
+import subprocess
+import time
+import os
+import signal
+
+def main(args):
+    recording_name = ''
+    method = args.input
+    recording_name = './sdcard/test.mp4'
+    if args.output is not None:
+            recording_name = './sdcard/' + args.output
+    if method is 'touch' and args.coordinate_x is None or args.coordinate_y is None:
+        print('USE: -i touch --coordinate-x <coordinate> --coordinate-y <coordinate> to use the touch input')
+        sys.exit()
+    elif method is 'intent':
+        record_with_intent(recording_name)
+    else:
+        simulate_input(args.coordinate_x, args.coordinate_y, recording_name)
+        exit()
+
+
+def record_with_intent(recording_name):
+    record_process = subprocess.Popen(['adb', 'shell','screenrecord'] + [recording_name])
+    time.sleep(3)
+    activity_start = subprocess.Popen(['adb', 'shell', 'am', 'start-activity', 'org.mozilla.fenix.nightly/org.mozilla.fenix.HomeActivity', '--ez finishonboarding true'])
```

Comment:
This intent differs between the two builds and is especially incorrect for our April builds (which should use the `fennecNightly` variant and the app ID/intent is `org.mozilla.fennec_aurora/.App`). I'd just delete this code because it's not working so we don't want to imply it is.

Reply:
That's odd... it worked withe april 20th Nightly from taskluster 🤔

Reply:
It works but it doesn't trigger the same user flow as users experience – `...HomeActivity` launches the HomeActivity directly while `...App` launches MigrationActivity which then opens HomeActivity.